test/testground/network/follower.go

```diff
@@ -87,7 +87,7 @@ func (f *Follower) Plan(ctx context.Context, runenv *runtime.RunEnv, initCtx *ru
 
 	runenv.RecordMessage("follower waiting for start height")
 
-	_, err = f.cctx.WaitForHeightWithTimeout(int64(10), time.Minute*7)
+	_, err = f.cctx.WaitForHeightWithTimeout(int64(5), time.Minute*7)
```

Comment:
[Question] Is the starting height value arbitrary, or is it determined by a specific logic within a part of the code? I'm curious about the rationale behind its change from `10` to `5.`

Reply:
iirc this was just an optimization as there's no benefit to waiting for block height 10. 5 is still a bit aribitrary

Reply:
I see, then why not 1?

Reply:
I don't have data to know how long this takes, but it does take some amount of time to establish connections between all the peers, so I guestimated 5